lib/activities/translate.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var assert = require("assert");
+
+var clone = require("clone"),
+    gdal = require("gdal");
+
+var output = require("./output"),
+    shell = require("./shell");
+
+module.exports = function translate(localInputPath, outputUri, options, callback) {
+  return output(outputUri, callback, function(err, localOutputPath, done) {
+    if (err) {
+      return callback(err);
+    }
+
+    var args = [];
+    if(options.outputFormat) {
+      args = args.concat(["-of", options.outputFormat]);
+    }
+
+    if(options.bands) {
+      var len = options.bands.length;
+      for (var i = 0; i < len; i++) {
+        args = args.concat(["-b", String(options.bands[i])]);
```

Comment:
It wasn't being coerced when the command was being executed, which is why I put that in there. Maybe I should check out what shell is doing and change that